Adamant/Modules/Wallets/Bitcoin/BtcWalletService+DynamicConstants.swift

```diff
@@ -75,8 +75,8 @@ extension BtcWalletService {
     
     static var nodes: [Node] {
         [
-            Node.makeDefaultNode(url: URL(string: "https://btcnode1.adamant.im")!, altUrl: URL(string: "http://176.9.38.204:44099")),
-Node.makeDefaultNode(url: URL(string: "https://btcnode3.adamant.im")!, altUrl: URL(string: "http://195.201.242.108:44099")),
+            Node.makeDefaultNode(url: URL(string: "https://btcnode1.adamant.im/bitcoind")!, altUrl: URL(string: "http://176.9.38.204:44099/bitcoind")),
+Node.makeDefaultNode(url: URL(string: "https://btcnode3.adamant.im/bitcoind")!, altUrl: URL(string: "http://195.201.242.108:44099/bitcoind")),
         ]
```

Comment:
Let's update adamant-wallets separately. Maybe there are some fixes required for the new nodes addresses